### DISCUSSION TITLE
Recommended feature flag framewrok/library

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#16947] [Q&A]
We want to implement a strategy of developing an app with feature flags.
What is the recommended approach for that?

**Comment by agneszitte** [comment]
Hi @xperiandri did you already take a look at our documentation related to feature flags?
https://platform.uno/docs/articles/feature-flags.html

If you have additional questions or think that we are missing some details in the current documentation please let us know.

(cc @jeromelaban, @MartinZikmund, @Youssef1313)

**Comment by kazo0** [comment]
@xperiandri You can also take a look at how Uno is doing feature flags here: https://github.com/unoplatform/uno/blob/fc86476f8fa3ae5158316a3269d962d8b2751b4e/src/Uno.UI/FeatureConfiguration.cs#L15
